includes/Modules/Analytics_4/Report/Request.php

```diff
@@ -395,6 +395,12 @@ protected function parse_dimension_filter( $dimension_name, $dimension_value ) {
 		$filter            = new $filter_class();
 		$filter_expression = $filter->parse_filter_expression( $dimension_name, $dimension_value );
 
+		if ( isset( $dimension_value['notExpression'] ) ) {
```

Comment:
`isset` will be true for any value but `null`. This should be `! empty` to ensure it is set and truthy.